pimd/pim_sock.c

```diff
@@ -315,6 +315,61 @@ int pim_socket_join(int fd, struct in_addr group, struct in_addr ifaddr,
 	return ret;
 }
 
+int pim_socket_leave(int fd, struct in_addr group, struct in_addr ifaddr,
```

Comment:
So why is it appropriate to just cut-n-paste the pim_socket_join function and just change IP_ADD_MEMBERSHIP to IP_DROP_MEMBERSHIP?

Reply:
Fixed